typescript/infra/config/environments/mainnet3/warp/configGetters/getEverclearConfig.ts

```diff
@@ -0,0 +1,341 @@
+            return [
+              destChain,
+              {
+                fee: parseInt(feeData?.tokenFee || '0'),
+                deadline: parseInt(feeData?.deadline || '0'),
+                signature: feeData?.sig || '',
+              },
+            ];
+          }),
+      );
+
+      const config: HypTokenRouterConfig =
+        asset === 'WETH'
+          ? {
+              owner,
+              mailbox: routerConfig[chain].mailbox,
+              type: TokenType.ethEverclear,
+              wethAddress: tokenAddresses[chain],
+              everclearBridgeAddress: everclearAdapterAddresses[chain],
+              outputAssets,
+              everclearFeeParams,
+            }
+          : {
+              owner,
+              mailbox: routerConfig[chain].mailbox,
+              type: TokenType.collateralEverclear,
+              token: tokenAddresses[chain],
+              everclearBridgeAddress: everclearAdapterAddresses[chain],
+              outputAssets,
+              everclearFeeParams,
+            };
+      return [chain, config];
+    }),
+  ) as ChainMap<HypTokenRouterConfig>;
+};
+
+export const getEverclearUSDCWarpConfig = async (
+  routerConfig: ChainMap<RouterConfigWithoutOwner>,
+  _abacusWorksEnvOwnerConfig: ChainMap<OwnableConfig>,
+  _warpRouteId: string,
+): Promise<ChainMap<HypTokenRouterConfig>> => {
+  return getConfigFromFeeData('USDC', routerConfig);
+};
+
+export const getEverclearETHWarpConfig = async (
+  routerConfig: ChainMap<RouterConfigWithoutOwner>,
+  _abacusWorksEnvOwnerConfig: ChainMap<OwnableConfig>,
+  _warpRouteId: string,
+): Promise<ChainMap<HypTokenRouterConfig>> => {
+  return getConfigFromFeeData('WETH', routerConfig);
+};
+
+export const getEverclearUSDTWarpConfig = async (
+  routerConfig: ChainMap<RouterConfigWithoutOwner>,
+  _abacusWorksEnvOwnerConfig: ChainMap<OwnableConfig>,
+  _warpRouteId: string,
+): Promise<ChainMap<HypTokenRouterConfig>> => {
+  return getConfigFromFeeData('USDT', routerConfig);
+};
+
+// Strategy configuration following CCTP pattern
+const safeChain = 'ethereum';
+const icaOwner = awSafes[safeChain];
+const safeSubmitter: SubmitterMetadata = {
+  type: TxSubmitterType.GNOSIS_SAFE,
+  chain: safeChain,
+  safeAddress: icaOwner,
+};
+
+const icaChains = Object.keys(awIcasLegacy);
+
+export const getEverclearStrategyConfig = (): ChainSubmissionStrategy => {
+  const submitterMetadata = EVERCLEAR_CHAINS.map((chain): SubmitterMetadata => {
+    if (!icaChains.includes(chain)) {
+      return {
+        type: TxSubmitterType.GNOSIS_SAFE,
+        chain,
+        safeAddress: awSafes[chain],
+      };
+    }
+
+    return {
+      type: TxSubmitterType.INTERCHAIN_ACCOUNT,
+      chain: safeChain,
+      owner: icaOwner,
+      destinationChain: chain,
+      internalSubmitter: safeSubmitter,
+    };
+  });
+
+  return Object.fromEntries(
+    EVERCLEAR_CHAINS.map((chain, index) => [
+      chain,
+      { submitter: submitterMetadata[index] },
+    ]),
+  );
+};
```

Comment:
seems to be the only place we reference the legacy icas here, but I dont see any ICA owners configured at all - should this be simplified?

we could consider using `awIcas` for all the non-ethereum chains and keeping only ethereum controlled by the awSafe?

Reply:
Done